src/test/java/com/starrocks/connector/spark/sql/CatalogTest.java

```diff
@@ -0,0 +1,63 @@
+// Copyright 2021-present StarRocks, Inc. All rights reserved.
```

Comment:
Please improve the test at least
1. verify the correctness
2. could run it using maven `mvn clean package -Dhttp_urls="your_fe_http" -Djdbc_urls="your_fe_jdbc"`

You can refer to `ReadWriteITTest` for an example

Reply:
fixed